susecloud

```diff
@@ -379,3 +379,12 @@ if [ -e /opt/dell/crowbar_framework/.crowbar-installed-ok ]; then
   find_and_plog_files_0 $chef_export_dir -type f
   rm -rf $chef_export_dir
 fi
+
+###############################################################
+section_header "RabbitMQ broker"
+if [ -e /usr/sbin/rabbitmqctl ]; then
+  rabbitmqctl report > /var/log/rabbitmq/rabbitmqctl_report.log
```

Comment:
You are right. I just copied it from ```crowbar_batch``` part without thinking about it.I'll fix it.